ci-operator/config/openshift/release/openshift-release-master__ci-4.9.yaml

```diff
@@ -208,6 +208,14 @@ tests:
     env:
       TEST_SUITE: openshift/conformance/serial
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-ovn
+  cron: '@daily'
```

Comment:
`interval: 24h`

Reply:
Some our tools don't understand `@daily` and won't be able to estimate how many historical records should be kept. Therefore `interval` is better than `cron`.

Reply:
@dmage done! thanks for the great suggestion :)